lib/packagers/index.js

```diff
@@ -36,7 +36,11 @@ const registeredPackagers = {
 module.exports.get = function (packagerId) {
   if (!_.has(registeredPackagers, packagerId)) {
     const message = `Could not find packager '${packagerId}'`;
-    this.serverless.cli.log(`ERROR: ${message}`);
+    if (this.log) {
+      this.log.error(`Could not find packager "${packagerId}"`);
```

Comment:
```suggestion
      this.log.error(message);
```

Reply:
I wrote it that way, so if we eventually would want drop support for old format, we can easily also drop `message` var.

Also quoting is different, in Framework convention is to use double quotes.

Anyway, I've updated up to your suggestion

Reply:
I didn't notice the quote change, I was focus on re-using the variable.
If the convention is to use double in Serverless, I'm fine with it!